lib/utils.js

```diff
@@ -399,6 +399,12 @@ exports.canonicalize = function canonicalize(value, stack, typeHint) {
         break;
       }
     /* falls through */
+    case 'module':
```

Comment:
With a bit more manual testing (when trying to reproduce) I think this issue may fundamentally occur anytime you are dealing with null-prototyped instances...

Which is an overall rare occurrence, but maybe this isn't the right approach?

e.g.

```
const foo = Object.create(null, { 
    [Symbol.toStringTag]: { value: 'Foo' }, 
     bing() { get: () => 'bong' } 
});

Object.prototype.toString.call(x); // '[object Foo]'

foo + ''; // Uncaught TypeError: Cannot convert object to primitive value
```
So while this solution works for _modules_, I don't think it actually solves the underlying issue